app/renderer/lib/webview-helpers.js

```diff
@@ -82,6 +82,11 @@ export function parseSource (source) {
   const scripts = $('script');
   scripts.remove();
 
-  // remove all `data-appium-desktop-` prefixes
-  return $.html().replace(/data-appium-desktop-/g, '');
+  return $.html()
+    // Remove all html entity / special chars that could break the parsing
+    .replace(/(&(.*?);)/ig, '')
```

Comment:
I would still prefer to avoid using regexps over dom traversal

Reply:
Agree with that, but wonder how I could walk over the dom and remove that from the content, need to dive into this

Reply:
the $ object is a [cheerio](https://github.com/cheeriojs/cheerio) parser instance. You could check https://stackoverflow.com/questions/21003384/how-to-remove-all-attributes-from-html for example on how to remove attributes from a document

Reply:
@mykola-mokhnach 

I used cheerio and htmlparser2 to make it safer, can you take a look again?